### DISCUSSION TITLE
Linux Mint's i3: Why Does "Ctrl + `" Keybinding Show "Connection Established"?

### DISCUSSION THREAD

**Discussion by SamuelBanya** [i3/i3#5692] [Q&A]
Hey there,

Is there any way to disable this particular keybinding? I think this might be 'Dunst' but I have no idea.
I'm using "Ctrl + ` + v" for an Emacs keybinding for Vterm, so I don't need this default behavior to show my connection status for my wired ethernet connection.

Thanks!

**Comment by okraits** [comment]
I guess you could bind it in i3 to `nop` to disable it: https://i3wm.org/docs/userguide.html#_nop
I'm not sure if it interferes with your Emacs keybinding then, though.

**Reply by orestisfl** [reply]
That would mean i3 consumes the event and emacs will not know

**Comment by orestisfl** [comment]
If the popup looks like dunst, check `/etc/dunst/dunstrc` for any interfering bindings.

**Comment by kfurkanbek** [comment]
This issue was bothering me for few days now.
I wanted to the same shortcut for "Switch Terminal" action in VSCode.

First, I checked the `/etc/dunst/dunstrc` path but my dunst was not located there.

1. So I used:
`find / -name dunst 2>/dev/null`
Then the listed locations were:
`/etc/xdg/dunst`
`/usr/share/doc/dunst`
`/usr/bin/dunst`

2. In `/etc/xdg/dunst/`
I edited the configuration file:
`gedit /etc/xdg/dunst/dunstrc`
I searched for `grave` then edited this:
`history = ctrl+grave` to `# history = ctrl+grave`

3. Saved and restarted the computer.

Now the `mod+grave` key is all mine haha ^^
